data/json/items/armor/ballistic_armor.json

```diff
@@ -845,31 +842,28 @@
       {
         "id": "fema_armor_vest",
         "name": { "str": "humanitarian ballistic vest" },
-        "description": "Bright blue in color this version was used by volunteers and officials from humanitarian groups.  Has \"FEMA\" written on the front lapel and across the back.",
-        "weight": 1,
-        "append": true
+        "description": "Bright blue in color, this slim armor vest was used by volunteers and officials from humanitarian groups during the first days of the Cataclysm before order broke down completely.  \"FEMA\" is written on the front lapel and across the back.",
+        "weight": 1
       },
       {
         "id": "police_armor_vest",
         "name": { "str": "police ballistic vest" },
-        "description": "Charcoal colored this vest was issued to police officers.  Has \"POLICE\" written on the front lapel and across the back.",
-        "weight": 1,
-        "append": true
+        "description": "This charcoal-colored plate carrier was once a common sight on police officers, and it provides adequate protection against small-caliber firearms in addition to featuring pouches to hold armor inserts.  \"POLICE\" is written on the front lapel and across the back.",
```

Comment:
plate carrier usually in my experience has a connotation of a simpler lighter weight armor vest.

perhaps "body armor" would be a better fit?